jetty9/src/main/docker/50-jetty.bash

```diff
@@ -0,0 +1,53 @@
+# Configure the environment and CMD for the Jetty Container
```

Comment:
There's quiet a bit of logic in this script and we need some kind of test coverage for it. I think it's possible to do it through the structure command tests. If you want to leave it to a separate PR, please file an issue for it.